defaults/main.yml

```diff
@@ -1560,8 +1718,70 @@ owncloud__apt_preferences__dependent_list_optional:
     pin: 'origin "download.owncloud.org"'
     priority: 995
     by_role: 'debops.owncloud'
-    state: '{{ owncloud__deploy_state }}'
+    state: '{{ "present"
+               if (owncloud__variant in ["owncloud"] and
+                   owncloud__deploy_state == "present")
+               else "absent" }}'
+
+
+# .. envvar:: owncloud__mariadb__dependent_databases [[[
+#
+# Configuration of the database managed by the debops.mariadb_ role.
+owncloud__mariadb__dependent_databases:
 
+  - database: '{{ owncloud__database_map[owncloud__database].dbname }}'
+    state: '{{ "present" if (owncloud__deploy_state != "purged") else "absent" }}'
+
+    # ## https://docs.nextcloud.com/server/12/admin_manual/installation/system_requirements.html#database-requirements-for-mysql-mariadb
+    # ## Currently experimental and not enabled by default.
+    # encoding: 'utf8mb4'
+    # collation: 'utf8mb4_general_ci'
+
+    # mariadb_server__options:
+    #   ## https://docs.nextcloud.com/server/11/admin_manual/installation/system_requirements.html#database-requirements-for-mysql-mariadb
+    #   'innodb_large_prefix': 'on'
+    #   'innodb_file_format': 'barracuda'
+    #   'innodb_file_per_table': 'true'
```

Comment:
Perhaps this should be included in the Nextcloud section of the Getting Started guide?